go.mod

```diff
@@ -8,8 +8,11 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
 	github.com/golang/protobuf v1.4.0-rc.4
+	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334 // indirect
```

Comment:
these indirect dependencies are added by the `make compile-protos`. IMO we should just let them go in. What do you think, @jessicayuen?

Reply:
Yeah, these keep coming back. Makes sense, let's allow them going forward.